go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/mattn/go-runewidth v0.0.13 // indirect
 	github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
```

Comment:
I don't see this package referenced anywhere.  Is it just a dependency of MSAL for go?

Reply:
Yes - they use it for some pretty printing in some error messages.